lib/Corres_UL.thy

```diff
@@ -1298,6 +1298,16 @@ lemmas corres_split_noop_rhs2
 
 lemmas corres_split_dc = corres_split[where r'=dc, simplified]
 
+lemma corres_add_noop_rhs:
```

Comment:
Maybe I should use an eta-expanded form here, but these are just moved for DRefine/CRefine, so I wasn't sure whether they should be modified (I did rename the schematics in `corres_add_noop_rhs`, but the rule is not currently being used with manually instantiated schematics)

Reply:
If we go to the trouble of making the PR, let's give it a shot. Doing the eta-expanded form could break some existing proofs (by renaming the variables it produces), but it'll be nicer and more stable going forward.

Reply:
Actually, now that I say this, I don't think we need it here: the rule shouldn't produce a new name, because it's not binding anything (the `_` makes sure the binding is unused).